test/runtests.jl

```diff
@@ -61,6 +64,14 @@ using Test
     @test ith_betti_number( H5, 4 ) == 1
     @test nr_of_q_rational_points( H5, 1 ) == 4
     
+    # Construct delPezzo surfaces
+    del_pezzo( -1 )
```

Comment:
There should be a way to test that this throws an error. In Oscar I saw something called `@test_throws` with a quick grep.